### ISSUE TITLE
Windows::Functions in win32-eventlog and win32-service have conflicting CreateEvent

### ISSUE THREAD

**Issue by btm** [djberg96/win32-eventlog#20]
win32-eventlog 0.6.5 changed the CreateEvent attachment function types from to `[:pointer, :int, :int, :string]` in the `Windows::Functions` module. win32-service 0.8.7 also has a `Windows::Functions` module that attaches CreateEvent but with bools instead.

Because these functions are attached in the same namespace, `Windows::Functions` they overwrite. When win32-eventlog is loaded by win32-service, you get the more recent types which win32-service hasn't been updated for and you get a type error, breaking win32-service.

```
C:/opscode/chef/embedded/lib/ruby/gems/2.0.0/gems/win32-service-0.8.7/lib/win32/daemon.rb:245:in `CreateEvent': no implicit conversion of true into Integer (TypeError)
```

I'd recommend each of the win32 projects move to unique namespace for `Windows::Functions`.

**Comment by djberg96** [comment]
Yep, they should be in their own namespace. Good catch, thanks.

**Comment by djberg96** [comment]
@btm I released win32-service 0.8.8 which should eliminate this issue. The win32-eventlog code still needs updating, too, but at these those two won't conflict.

**Comment by djberg96** [comment]
Fixed in 0.6.6. Thanks for the report.
